tests/langchain/test_langchain_model_export.py

```diff
@@ -1234,7 +1234,7 @@ def _llm_type(self) -> str:
         assert response3[i]["choices"][0]["message"] == expected_resp_content
 
 
-def tests_save_load_complex_runnable_parallel():
+def test_save_load_complex_runnable_parallel():
```

Comment:
It's possible to configure pytest to run functions starting with `tests_` as well but I think https://github.com/mlflow/mlflow/pull/12864 is a better option.